src/playbacks/no_op/no_op.js

```diff
@@ -25,13 +25,16 @@ export default class NoOp extends Playback {
     this.$el.html(this.template({message: playbackNotSupported}))
     this.$el.append(style)
     this.trigger(Events.PLAYBACK_READY, this.name)
-    this._options.autoPlay && this.play()
+    const showForNoOp = !!(this.options.poster && this.options.poster.showForNoOp);
+    if (this.options.autoPlay || !showForNoOp) {
+      this.play()
+    }
```

Comment:
I didn't realise it used up that much. Maybe we could make the noise generation more efficient?